src/doc/trpl/references-and-borrowing.md

```diff
@@ -125,6 +125,10 @@ This will print `6`. We make `y` a mutable reference to `x`, then add one to
 the thing `y` points at. You’ll notice that `x` had to be marked `mut` as well,
 if it wasn’t, we couldn’t take a mutable borrow to an immutable value.
 
+You'll also notice we added an asterisk in front of `y`, making it `*y`,
+this is because y is an `&mut` reference. You'll also need to use them for
+accessing and modifying `&` references as well.
```

Comment:
`&T` types can't be modified, I would just say "accessing the contents of a reference", maybe?